x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/connector/connector_configuration.tsx

```diff
@@ -285,7 +285,7 @@ service_type: "${index.connector.service_type || 'changeme'}"
                       </EuiFlexItem>
                     </EuiFlexGroup>
                   ),
-                  status: index.connector.scheduling.enabled ? 'complete' : 'incomplete',
+                  status: index.connector.scheduling.full.enabled ? 'complete' : 'incomplete',
```

Comment:
question: Is ent-search going to migrate old connector documents to the new format? Or do we need to be backwards compatible here?

Reply:
Yes @vidok was writing some migration for them.